docs/development.md

```diff
@@ -384,3 +385,79 @@ If you are firing a conversion event for a specific test, be sure that the `part
 |   `Prod`  | `false` (real user) |  `Prod`  | `Prod`   |       `Prod`      | `Prod`  |
  
 
+## 10 CSS guidelines
+
+Currently in `support-frontend` there are two types of components. 
+
+Shared component: Components that are used in more than one page, they are located inside the global `components` folder.
```

Comment:
Perhaps add:

These should only contain rules which are context-independent. For instance, font sizes, colours, padding (since it's internal), margins of child elements. But margin on the parent, for example, should probably be avoided, because it's presupposing that it will be used in a particular place. This kind of positional styling should happen in the page-level CSS